asyncpg/connection.py

```diff
@@ -126,6 +127,24 @@ def __init__(self, protocol, transport, loop,
             del self._listeners[channel]
             await self.fetch('UNLISTEN {}'.format(channel))
 
+    def add_notice_callback(self, callback):
+        """Set a callback to be used when asyncronous NoticeResponse is received
```

Comment:
The first line of the docstring should be a complete sentence describing what the method does.

Then an empty new line.

Then you can add more paragraphs.